### PR TITLE
Issue 226 - Move Validate and Save menu

### LINKED ISSUE #226: Move Validate and Save menu
Move the Validate and save buttons to the bottom of each component.

### PR DESCRIPTION
**Pull request by Rachel-ais**
closes #226

### CODE REVIEW

**Comment by mikenholt** [comment]
Observations: 
- Recipe Validate/Save not working
- Popup about navigating away from page occurring on save: Strikes, Workspaces, ~Scans~, (have not tested jobs and recipes)

**Comment by mikenholt** [comment]
Investigate why the recipe-type is not being validated on click.  The function always returns `true` with a form that is missing required values.

**Comment by Rachel-ais** [comment]
I think the pop ups should be fixed

File: projects/developer/src/app/configuration/recipe-types/component.ts [hunk 1]
```diff
@@ -459,14 +461,16 @@ export class RecipeTypesComponent implements OnInit, OnDestroy {
...
         }, err => {
             console.log(err);
             this.messageService.add({ severity: 'error', summary: 'Error validating recipe type', detail: err.statusText });
         });
     }
 
     saveRecipeType() {
+        console.log(this.isSaving);
+        this.isSaving = true;
+        console.log(this.isSaving);
```
Comment (mikenholt) [inline projects/developer/src/app/configuration/recipe-types/component.ts]: Remove print statements?

Comment (Rachel-ais) [reply projects/developer/src/app/configuration/recipe-types/component.ts]: oops sorry!

**Pull request merged**